tox.ini

```diff
@@ -20,13 +21,31 @@ commands =
 
 [testenv:type]
 skipsdist = true
-skip_install = true
+setenv =
+    DJANGO_CONFIGURATION = DevelopmentConfiguration
+passenv =
```

Comment:
Also, Tox has a better syntax to copy values from another environment: https://tox.wiki/en/latest/config.html#substitution-for-values-from-other-sections

Use that, instead of repeating yourself.

Reply:
Seems like everything is working properly now without this. But thanks, that's good to know :+1: